src/constants.lua

```diff
@@ -28,12 +44,11 @@ constants.gatewayOperatorRewardRatio = 0.9
 
 -- GAR
 constants.DEFAULT_UNDERNAME_COUNT = 10
-constants.DEADLINE_DURATION_MS = constants.oneHourMs
 constants.totalTokenSupply = constants.ARIOToMARIO(1000000000) -- 1 billion tokens
 constants.MIN_EXPEDITED_WITHDRAWAL_PENALTY_RATE = 0.10 -- the minimum penalty rate for an expedited withdrawal (10% of the amount being withdrawn)
 constants.MAX_EXPEDITED_WITHDRAWAL_PENALTY_RATE = 0.50 -- the maximum penalty rate for an expedited withdrawal (50% of the amount being withdrawn)
-constants.minimumWithdrawalAmount = constants.mARIOPerARIO -- the minimum amount that can be withdrawn from the GAR
-constants.redelegationFeeResetIntervalMs = constants.defaultEpochDurationMs * 7 -- 7 epochs
+constants.minimumWithdrawalAmount = constants.ARIOToMARIO(1) -- the minimum amount that can be withdrawn from the GAR
+constants.redelegationFeeResetIntervalMs = constants.daysToMs(7) -- 7 days
```

Comment:
Note: this was previously denominated in epochs so you might want to use a constant or function that deals in that denomination.

Reply:
delegation fees have no affiliated to epochs